ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -140,6 +144,32 @@ static void bootstrapOrUpgrade(String hardwareLayoutPath, String partitionLayout
     clusterMapToHelixMapper.logSummary();
   }
 
+  /**
+   * Takes in the path to the files that make up the static cluster map and uploads the cluster configs(such as partition
+   * override) to HelixPropertyStore in zookeeper.
+   * @param hardwareLayoutPath the path to the hardware layout file.
+   * @param partitionLayoutPath the path to the partition layout file.
+   * @param zkLayoutPath the path to the zookeeper layout file.
+   * @param clusterNamePrefix the prefix that when combined with the cluster name in the static cluster map files
+   *                          will give the cluster name in Helix to bootstrap or upgrade.
+   * @param maxPartitionsInOneResource the maximum number of Ambry partitions to group under a single Helix resource.
+   * @param helixAdminFactory the {@link HelixAdminFactory} to use to instantiate {@link HelixAdmin}
+   * @throws IOException if there is an error reading a file.
+   * @throws JSONException if there is an error parsing the JSON content in any of the files.
+   */
+  static void uploadClusterConfigs(String hardwareLayoutPath, String partitionLayoutPath, String zkLayoutPath,
+      String clusterNamePrefix, int maxPartitionsInOneResource, HelixAdminFactory helixAdminFactory) throws Exception {
+
+    HelixBootstrapUpgradeUtil clusterMapToHelixMapper =
+        new HelixBootstrapUpgradeUtil(hardwareLayoutPath, partitionLayoutPath, zkLayoutPath, clusterNamePrefix,
+            maxPartitionsInOneResource, false, false, helixAdminFactory);
+    Map<String, Map<String, String>> partitionOverrideInfos =
+        clusterMapToHelixMapper.generatePartitionOverrideFromAllDCs();
+    info("Uploading partition override to HelixPropertyStore based on static clustermap.");
```

Comment:
we should stop using the term "static clustermap" especially for things that are newly introduced.

Reply:
Right, I will change it to avoid using term `static clustermap`